doc/source/tune/tutorials/tune-output.rst

```diff
@@ -149,8 +149,28 @@ respectively:
 The file names are relative to the trial's logdir. You can pass absolute paths,
 too.
 
-If ``log_to_file`` is set, Tune will automatically register a new logging handler
-for Ray's base logger and log the output to the specified stderr output file.
+Caveats
+^^^^^^^
+Logging that happens in distributed training workers (if you happen to use Ray Tune together with Ray Train)
+is not part of this ``log_to_file`` configuration.
+
+Where to find ``log_to_file`` files?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If your Tune workload is configured with syncing to head node, then the corresponding ``log_to_file`` outputs
+can be located under each trial folder.
+If your Tune workload is instead configured with syncing to cloud, then the corresponding ``log_to_file``
+outputs are *NOT* synced to cloud and can only be found in the worker nodes that the corresponding trial happens.
+
+.. note::
+    This can get ugly when the trainable is moved across different nodes throughout its lifetime like in some
+    scheduler's case. We may prioritize enabling this if there are enough user requests.
```

Comment:
```suggestion
    This can get ugly when the trainable is moved across different nodes throughout its lifetime. This can happen with some schedulers. We may prioritize enabling this if there are enough user requests. Please open an issue on Ray GitHub if this impacts you.
```